admin-manual/installation/linux/ubuntu-xenial.rst

```diff
@@ -360,24 +360,21 @@ Gearman job server is required by AtoM as of version 2.2.
 
 Other packages
 --------------
-
-In order to generate PDF finding aids, AtoM requires `Apache FOP 2.1 <https://archive.apache.org/dist/xmlgraphics/fop/binaries/fop-1.0-bin.tar.gz>`__.
-After downloading and extracting it, ensure you have the fop executable in your
-system's executable path. Additionally, you may need to set the environmental
-variable FOP_HOME to the folder path you extracted Apache FOP. The following is
-a single command that will set up everything for you automatically:
+In order to generate PDF finding aids, AtoM requires Apache FOP 2.1 to be
+installed. Fortunately, Apache FOP 2.1 can now be installed directly from
+Ubuntu packages using the command below.
+
+The command specified below uses the '--no-install-recommends' parameter:
```

Comment:
I think you might want to use ` `` `  (double ticks) in sphinx around code or CLI options, to display them as raw.